sphinx/source/quickstart/index.rst

```diff
@@ -3,7 +3,24 @@ Quickstart
 
 First, you should :ref:`setup a CCF-compatible environment <Environment Setup>`. Then, you will be able to :ref:`build CCF from source and run CCF test suite <Building CCF>`. Note that for rapid prototyping, you can run a `virtual` build of CCF that does not require Intel SGX.
 
-Once this is done, make sure you get familiar with some of :ref:`CCF concepts`. You will then be able to:
+Once this is done, you can quickly spin up a CCF network and start :ref:`issuing commands to the deployed application <Issuing Commands>`:
+
+.. code-block:: bash
+
+    $ cd CCF/build
+    $ ../deploy_ccf.sh libloggingenc
```

Comment:
I don't like `deploy`, as it suggests like a _real_ network rather than a temporary test thing. Maybe `start_network`, or `start_test_network`, or `launch_test_instance`, or...

Reply:
`start_test_network.sh` it is 👍